### PR TITLE
Ember Class Pirate Raider

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
This ship was once used for exploration when settlers first entered the sector. Though it has been many rotations since this model last glided through the empty vacuum of space. While most of them lay abandoned, or in scrapyards, one of them has managed to escape this faith. This ship, The Cutter, is now the home of a particular group of Frontiersmen. While the ship has seen better days, its size is more than enough to strike fear into any merchant ship unlucky enough to enter its domain. 

so! let's get into what this ship is, and what it does. 
the ship is used by the Frontiersmen, as a show of force, raiding, and ship-to-ship combat. on the front, it has tw- one gun bay, capable of launching explosives (or people) at whatever foolishly somehow places itself in front of the barrel. it has a decently stocked armory, a makeshift maintenance chembay, a medbay in disrepair, a bar/kitchen/botany, a destroyed comms array, a small cargo bay, and a pretty massive engineering bay. The ship is powered by its own state-of-the-art "tm" Supermatter. 

the ship is as pirate ships are, an admin spawn. Its intent is to serve as an antagonist in rounds, something that could create some interesting, and fun roleplay. 

the crew:
1x Captain
2x Bridge Officer
1x Head of Security (Master At Arms)
2x Cargo Tech
1x Medical Doctor (Surgeon)
2x Engineer (Carpenter)
1x Atmos Tech
1x Chief Engineer (Head Carpenter)
1x Cook (Steward)
3x Assistant (Deckhand)
3x security officer (Buccaneer)
Ship images:

image

image
(don't mind the broken table)

image

image

image

image

## Why It's Good For The Game
currently, there isn't much Frontiersmen content, nor pirate content in the codebase despite them being one of the only real antagonists we have. This is one of the ships I've made for them, and one I think could do well in creating some fun RP

## Changelog

:cl:
add: adds the pirate_cutter
add: adds the scar suit
add: misc frontiersmen jobs
/:cl:

### CODE REVIEW

**Comment by goober3** [comment]
the roles don't have Names in the json. and the assistant role name should probably be Gender Neutral. also Maybe add more Murder Roles???? cause it's a pirate ship!

**Comment by Apogee-dev** [comment]
Give your job titles more flavor, they're pirates, not a corporation. They don't have "assistants" they have deckhands, grunts, spacers, whatever you pick. Maybe look at the libertatia for some pirate-y ideas.

File: _maps/configs/pirate_cutter.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+			"officer": true,
+			"slots": 1
+		},
+		"Cook": {
+			"outfit": "/datum/outfit/job/cook/frontiersmen",
+			"slots": 1
+		},
+		"Assistant": {
+			"outfit": "/datum/outfit/job/assistant/frontiersmen",
+			"slots": 6
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: remove 4 assistant slots, refluff as an expendable boarding crew job.

File: _maps/configs/pirate_cutter.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			"slots": 1
+		},
+		"Chief Engineer" :{
+			"outfit": "/datum/outfit/job/ce/frontiersmen",
+			"officer": true,
+			"slots": 1
+		},
+		"Cook": {
+			"outfit": "/datum/outfit/job/cook/frontiersmen",
+			"slots": 1
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: i feel like cook should be an assistant job.

File: _maps/configs/pirate_cutter.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		},
+		"Engineer": {
+			"outfit": "/datum/outfit/job/engineer/independent/frontiersmen",
+			"slots": 2
+		},
+		"Atmospheric Technician": {
+			"outfit": "/datum/outfit/job/atmos/frontiersmen",
+			"slots": 1
+		},
+		"Chief Engineer" :{
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: move this towards other nominally command roles at the top of the file

File: _maps/configs/pirate_cutter.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		},
+		"Bridge Officer": {
+			"outfit": "/datum/outfit/job/head_of_personnel/frontiersmen",
+			"officer": true,
+			"slots": 2
+		},
+		"Head Of Security": {
+			"outfit": "/datum/outfit/job/hos/frontiersmen",
+			"officer": true,
+			"slots": 1
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: re-theme for boarding.

**Comment by Erikafox** [comment]
where are the other pirate ships, spockye

File: _maps/configs/pirate_cutter.json (hunk 1 above)
Comment (spockye) [reply _maps/configs/pirate_cutter.json]: the entire crew serves as the "combat crew" like it usually was on pirate ships. i can change this if you think its necessary

File: _maps/configs/pirate_cutter.json (hunk 2 above)
Comment (spockye) [reply _maps/configs/pirate_cutter.json]: i suppose. just wanted to add some flavour to the list

**Comment by spockye** [comment]
> the roles don't have Names in the json. and the assistant role name should probably be Gender Neutral. also Maybe add more Murder Roles???? cause it's a pirate ship!

ripped "cabin boy" from the renowned "pirate ship jobs" page, but alas. (also the entire crew is the murder crew)


> Give your job titles more flavor, they're pirates, not a corporation. They don't have "assistants" they have deckhands, grunts, spacers, whatever you pick. Maybe look at the libertatia for some pirate-y ideas.

all of them (outside of atmos tech) have custom jobnames.

**Comment by spockye** [comment]
> where are the other pirate ships, spockye

in my treasure trove. 5 sheckles per ship

**Comment by Erikafox** [comment]
>all of them (outside of atmos tech) have custom jobnames.

The names you define in the job datums is what they show up as in the outfit editor.
You need to define what you want them to be on the ship in the ships config.

File: _maps/configs/pirate_cutter.json [hunk 5]
```diff
@@ -0,0 +1,60 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Cutter-Class Pirate Raider",
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: cutter is a REALLY generic name and also a class of boat. I'd figure something else out.

File: _maps/configs/pirate_cutter.json [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+	"prefix": "ISV",
```
Comment (Erikafox) [inline _maps/configs/pirate_cutter.json]: Ask loreheads about frontiersman signaling + admins about IFF false flagging

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 7]
```diff
@@ -567,3 +567,15 @@ Contains:
 	armor = list("melee" = 40, "bullet" = 20, "laser" = 20,"energy" = 30, "bomb" = 60, "bio" = 100, "rad" = 90, "fire" = 85, "acid" = 75)
 	slowdown = 0.5
 	w_class = WEIGHT_CLASS_NORMAL
+
+
+/obj/item/clothing/suit/space/scar
```
Comment (Erikafox) [inline code/modules/clothing/spacesuits/miscellaneous.dm:572]: No paired helmet?

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
Put a windoor here so you can't accidentally slip in.
image
I'm not huge on load-in block chance weapons, Or vibroweapons honestly. Get a different sprite - path a more nautical sword of some kind?
image
Set a GPS tag and rename the megaphone.
StrongDMM_anz4EKXV19
The armory should not start open.
Why is there a knife on the ground?
StrongDMM_NpSqwB2yTV
rename the microphone, set the iconstate on the wideband to "intercom-wideband-table"
StrongDMM_VXVSTy6jsz
rename the gas mask in the MOA's locker.
image
Bad spot for an autolathe, the terminal will be under it.
StrongDMM_k22kIVUXr9
While funny, the zero days since last accident sign doesn't really //fit//
don't have a fully cycling airlock there either, it seems like recent damage given the state of the hull, so they shouldn't have the thing there.
rename the rocks at the top half.
StrongDMM_00ZelCmNaC
This area needs to be pathed as cargo.
StrongDMM_ZqMEAdXx0b
Innovative, unfortunately you cannot use them on the pipe until they are dried. Put a drying rack and maybe some seeds somewhere?
StrongDMM_SRxLVxpl5N
The launch button is far too hidden.
StrongDMM_lNLoviBVkE
I don't think this ship needs a DNA manipulator. I'd say it needs a portable seed thingy but I'm liking what overmap 4.2 is going to do for that.

**Comment by Erikafox** [comment]
almost forgot.

It just doesn't seem very aerodynamic, could you do something about that?

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 7 above)
Comment (spockye) [reply code/modules/clothing/spacesuits/miscellaneous.dm:572]: looks too memorable combined with the helmet.

**Comment by Erikafox** [comment]
fix the pipes

**Comment by Erikafox** [comment]
fix the pipes

**Comment by goober3** [comment]
> fix the pipes

i'm not apologizing for this!

**Comment by Erikafox** [comment]
image
I've decided I don't like the SM. Please figure out an alternative power source

**Comment by spockye** [comment]
> image I've decided I don't like the SM. Please figure out an alternative power source

the ship was designed on having one. i'd rather not do that thank you

**Comment by AverageUser67** [comment]
how does a pirate ship covered in rust get an sm shard if they're so prohibitively expensive/rare that they haven't been weaponized yet

**Comment by spockye** [comment]
Stealing
Looting
Some might even call it plundering

**Comment by AverageUser67** [comment]
how the fuck did they steal a high-value unstable product with no engine

**Comment by meemofcourse** [comment]
they walked in and took it. simple as

**Review by Erikafox — CHANGES_REQUESTED** [review]
something something look at SM alternatives and put an exterior docking port on

**Review by Erikafox — APPROVED** [review]
other maptainers arise from your slumber and review this too please thanks

**Review by Latentish — APPROVED** [review]
I've seen this in game, I've tested it, it's good. This is my approval.

**Pull request merged**